src/index.py

```diff
@@ -2,11 +2,30 @@
 
 app = Flask(__name__)
 
+changed_events = [
+    {
+        'Name': 'Dekmantel',
+        'Date': '25-04-2021',
+        'Website': 'www.dekmantel.com'
+    },
+    {
+        'Name': 'Event2',
+        'Date': '25-04-2021',
+        'Website': 'www.event2.com'
+    }
+]
+
+
+
 
 @app.route('/')
 def index():
     return render_template("index.html")
 
+@app.route('/mail')
```

Comment:
This is nice for testing and taking a look at the email but it's not needed to actually send the emails. You can leave it there for now, we'll work on the emails again later.